UnityProject/Packages/packages-lock.json

```diff
@@ -252,7 +252,7 @@
       "depth": 0,
       "source": "git",
       "dependencies": {},
-      "hash": ""
```

Comment:
Can you make sure this is empty since if it generates a hash it always pulls the same version of that, and you have to empty it out to get the newest version